### PR TITLE
Fix EBF calculating twice and Oil Cracker having wrong discount

### PR DESCRIPTION
**Pull request by BlueWeabo**

### CODE REVIEW

File: src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_OilCracker.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ protected ProcessingLogic createProcessingLogic() {
             protected GT_OverclockCalculator createOverclockCalculator(@Nonnull GT_Recipe recipe,
                 @Nonnull GT_ParallelHelper helper) {
                 return super.createOverclockCalculator(recipe, helper)
-                    .setEUtDiscount(Math.max((0.1F * (heatLevel.getTier() + 1.0F)), 0.5F));
+                    .setEUtDiscount(1.0F - Math.min(0.1F * heatLevel.getTier(), 0.5F));
```
Comment (miozune) [inline src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_OilCracker.java:183]: Isn't it `0.1F * (heatLevel.getTier() + 1)`? It used to be
```java
// heatLevel.getTier() starts at 0
if (this.heatLevel.getTier() < 5) {
    this.mEUt *= 1 - (0.1D * (this.heatLevel.getTier() + 1));
} else {
    this.mEUt *= 0.5;
```

Comment (BlueWeabo) [reply src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_OilCracker.java:183]: Did Cupronickel coils also give the bonus? If so I will edit it  back.

**Review by miozune — COMMENTED** [review]
btw squash-merging PR with single commit saying "fix" will result in the merge commit being named as "fix" by default, no matter how you name the PR. Giving proper name to each commit is recommended.

**Comment by BlueWeabo** [comment]
Will Remember it for next time. I forget that happens like that

**Pull request merged**